src/test/java/gov/healthit/chpl/aqa/pageObjects/ChartsPage.java

```diff
@@ -23,5 +23,9 @@ public static WebElement chartsHealthITFeedback(final WebDriver driver) {
     public static WebElement chartsInfoText(final WebDriver driver) {
         return driver.findElement(By.xpath("//*[@id=\"main-content\"]/div[1]/div/p"));
     }
+
+    public static WebElement chartsPageTitle(final WebDriver driver, final String pageTitle) {
```

Comment:
This looks like the kind of definition that might fit in the Base Page class; there's nothing in this xpath that's specific to the Charts page